views.py

```diff
@@ -342,7 +342,7 @@ def events():
     for event in events_response.forwarding_events:
         tx_date = datetime.datetime.fromtimestamp(event.timestamp).strftime("%Y-%m-%d")
         tx_size = event.amt_out
-        tx_fee = event.fee
+        tx_fee = event.fee_msat / 1000
```

Comment:
here im not sure if this is correct in python, (i normaly programm in C/C++), but it works. ;)